mri/reconstruct/utils.py

```diff
@@ -162,7 +162,7 @@ def generate_operators(data, wavelet_name, samples, mu=1e-06, nb_scales=4,
     # Local imports
     from mri.numerics.cost import GenericCost
     from mri.numerics.linear import WaveletN, WaveletUD2
-    from mri.numerics.fourier import FFT2
+    from mri.numerics.fourier import FFT
```

Comment:
https://github.com/CEA-COSMIC/pysap-mri/blob/d1ac8ca10852d922f37bab51d56d6801bd6fde06/mri/reconstruct/utils.py#L181-L183

You may have to change this also. We will now start supporting the function

Reply:
Ok I'll do it,
We should also probably add an example with the cartesian 3D reconstruction with or without sensitivity maps estimation

Reply:
Maybe add it as a part of this PR itself?

Reply:
Working on it